CodePushPackage.m

```diff
@@ -228,8 +228,16 @@ + (void)downloadPackage:(NSDictionary *)updatePackage
             NSString * unzippedFolderPath = [CodePushPackage getUnzippedFolderPath];
             NSMutableDictionary * mutableUpdatePackage = [updatePackage mutableCopy];
             if (isZip) {
-                NSError *nonFailingError = nil;
+                if ([[NSFileManager defaultManager] fileExistsAtPath:unzippedFolderPath]) {
```

Comment:
Can you add a brief comment here explaining why you're doing this? It would be helpful to remember that this is for catching the scenario were a previous download failed. BTW, could you also confirm that this folder would get removed in the future if a new binary was discovered, as opposed to waiting for a new CodePush update?